tox.ini

```diff
@@ -19,11 +19,20 @@ commands =
   coverage run {toxinidir}/manage.py test
   coverage report --show-missing
 
-[testenv:flake8]
+[testenv:lint]
 deps =
   flake8
   flake8-docstrings
   flake8-import-order
   flake8-quotes
+  pipenv
+  pyflakes
```

Comment:
Does pyflakes need to be a specific dependency? It is only supported through Python 3.3

Reply:
removed.